src/services/dashboards/dashboard-detail/lib/themeHelper.ts

```diff
@@ -0,0 +1,31 @@
+import type { WidgetThemeOption } from '@/services/dashboards/dashboard-detail/lib/type';
+
+const WIDGET_THEME = ['violet', 'blue', 'coral', 'yellow', 'gray', 'green', 'indigo', 'peacock'] as const;
+const WIDGET_THEME_LEN = WIDGET_THEME.length;
+
+// ['violet' | 'blue' | 'coral' ... | undefined]
+export type WidgetThemeAssignedList = Array<typeof WIDGET_THEME[number] | undefined>;
```

Comment:
suggestion: THEME constant is already exist(yuda made it) so you can reuse it :)

Reply:
I'll apply it. Thanks !!!